backend/src/gh.rs

```diff
@@ -43,11 +43,11 @@ struct Claims {
 }
 
 impl Claims {
-    pub fn new() -> Result<Self> {
+    pub fn new(config: Arc<HydeConfig>) -> Result<Self> {
```

Comment:
```suggestion
    pub fn new(client_id: &str) -> Result<Self> {
```

How do you feel about making this take the Github client id directly, rather than passing in the entire config?

Then the code that calls `Claims::new` would access the config instead

Reply:
Good shout, will implement this